.github/workflows/benchmark.yaml

```diff
@@ -0,0 +1,36 @@
+name: Benchmarks
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  phpbench:
+    name: "PHPBench"
+    runs-on: "ubuntu-20.04"
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version:
+          - "5.4"
+          - "7.4"
+          - "8.2"
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

Comment:
should use `actions/checkout@v3` to avoid deprecations about the nodejs-12 runtime

Reply:
https://github.com/Masterminds/html5-php/pull/224